cobald_hep_plugins/stopper.py

```diff
@@ -0,0 +1,65 @@
+from cobald.interfaces import Pool, PoolDecorator
+
+from cobald.utility import enforce
+
+import asyncio
+
+from cobald.daemon import service
+
+
+@service(flavour=asyncio)
+class Stopper(PoolDecorator):
+    """
+    Decorator that sets demand to 0 if the partition has no pending jobs
```

Comment:
I think you can drop restricting this just to a partition from here and later on – the code/plugin is much more generally useful.
```suggestion
    Decorator that sets demand to 0 if there are no pending jobs
```